packages/driver/src/cypress/command.ts

```diff
@@ -136,6 +134,12 @@ export class $Command {
     return this
   }
 
+  pick (...args) {
+    args.unshift(this.attributes)
+
+    return _.pick.apply(_, args as [Record<string, any>, any[]])
+  }
```

Comment:
I don't necessarily think we need to be added this to every command instance. Why can't we use a static method?

Reply:
It's because of the line below:

https://github.com/cypress-io/cypress/blob/331e5412f566490e75d45d52d8111836745c336c/packages/driver/src/cypress/log.ts#L125